Cargo.toml

```diff
@@ -77,17 +69,16 @@ quote = "1.0.32"
 proc-macro2 = "1.0.66"
 cargo_metadata = "0.15"
 colored = "2"
+bincode = "2.0.0-rc.3"
```

Comment:
They are very very careful with their versioning, but 2.0.0 is out for a long time, they just want all issues marked `v2` to be fixed. So people seems just to use that. I'd rather not change it, v1 is using serde and has rough edges because of that.

Reply:
Ok, I hope we have dependapot set up to tell us when there's something more stable?

Reply:
I enabled dependabot